app/controllers/google/autocomplete_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Google::AutocompleteController < ApplicationController
+  def index
+    #OPEN SSL NEEDS TO BE FIXED PROPERLY
+
+    @client = GooglePlaces::Client.new(ENV['GOOGLE_KEY'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.